matching-service/index.js

```diff
@@ -1,17 +1,29 @@
-import express from 'express';
-import cors from 'cors';
-import { createServer } from 'http';
+import "dotenv/config";
+import express from "express";
+import cors from "cors";
+import { createServer } from "http";
+
+import router from "./api/routes.js";
+import { initIo } from "./socket.js";
+
+const PORT = process.env.PORT || 8001;
 
 const app = express();
-app.use(express.urlencoded({ extended: true }))
-app.use(express.json())
-app.use(cors()) // config cors so that front-end can use
-app.options('*', cors())
 
-app.get('/', (req, res) => {
-    res.send('Hello World from matching-service');
-});
+// start routes
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(cors()); // config cors so that front-end can use

```

Comment:
Should use the following instead, otherwise frontend will reject request.
`app.use(cors({ origin: true, credentials: true }));`